### ISSUE TITLE
Make a documentation of docker 

### ISSUE THREAD

**Issue by deming7h777** [NAL-i5K/general_issues#11]

**Comment by deming7h777** [comment]
https://github.com/NAL-i5K/notes

**Comment by hsiaoyi0504** [comment]
Hi @deming7h777, I would suggest following things:
1. make the note in plain English
2. make a file name without space
3. put a link of the note in README.md 
4. keep the root folder clean. That is, put this new documentation into a folder. Perhaps a new folder because previous notes are not pretty compatible to git.
5. Choose a markup language to format that document such as markdown (because it renders better on github) and make the filename with corresponding file extension.

**Comment by hsiaoyi0504** [comment]
@deming7h777 You accidentally deleted your documentation: https://github.com/NAL-i5K/notes/commit/3b0686cdc0311679adb545ff4f4de637a18b9b9a.

**Comment by hsiaoyi0504** [comment]
@deming7h777 Just a kindly reminder that it seems to me that the documentation is accidentally deleted https://github.com/NAL-i5K/notes/commit/3b0686cdc0311679adb545ff4f4de637a18b9b9a.

**Comment by deming7h777** [comment]
oh thank you so much !
I don't even notice that, I'll upload again.
